projects/experimental/components/badge-alert/badge-alert.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Inject,
+    Input,
+    ViewEncapsulation,
+} from '@angular/core';
+import {TUI_PLATFORM, TuiPlatform} from '@taiga-ui/cdk';
+import {
+    MODE_PROVIDER,
+    TUI_ANIMATION_OPTIONS,
+    TuiAnimationOptions,
+    tuiPop,
+    TuiSizeL,
+    TuiSizeXS,
+} from '@taiga-ui/core';
+
+@Component({
+    selector: 'tui-badge-alert',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    template: '<ng-content></ng-content>',
+    styleUrls: ['./badge-alert.style.less'],
+    encapsulation: ViewEncapsulation.None,
```

Comment:
What's the point of disabled encapsulation?

Reply:
just for easy color customization